.github/workflows/ansible.yml

```diff
@@ -104,13 +104,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.9", "3.10", "3.11"]
-        ansible-version: [stable-2.17, stable-2.15, stable-2.16]
-        exclude:
-        # Ansible-core 2.16 is supported only from Python 3.10 onwards
```

Comment:
maybe better to use `["3.10", "3.11", "3.12"]` in the matrix and remove those excludes?

Reply:
yes, maybe that would be better. changed it int the latest commit